WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift

```diff
@@ -24,16 +24,20 @@ class LoginEpilogueViewController: UIViewController {
         }
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         super.prepare(for: segue, sender: sender)
         if let vc = segue.destination as? LoginEpilogueTableView {
             tableViewController = vc
         }
     }
 
-    // @IBAction to allow to set the selector for target in the storyboard
-    @IBAction func unwindOut(segue: UIStoryboardSegue) {
```

Comment:
@aerych It looks like this deleted on WPTabBarController.h was missed. I'm getting a warning in Xcode:
![2017-06-09_13-07-06](https://user-images.githubusercontent.com/373903/26988339-8c8f3172-4d14-11e7-83b8-d6803c6ba2b3.png)